frameworks/Swift/vapor/Sources/vapor-tfb/ServerMiddleware.swift

```diff
@@ -0,0 +1,18 @@
+import Vapor
+
+/// Middleware that adds `Server` HTTP header to response.
+public final class ServerMiddleware: Middleware, ServiceType {
+
+    public init() { }
+    
+    public func respond(to request: Request, chainingTo next: Responder) throws -> Future<Response> {
+        return try next.respond(to: request).map { res in
+            res.http.headers.replaceOrAdd(name: .server, value: "Vapor")
```

Comment:
I think we are safe to use just `add` here since we know for sure no one else will be adding it. My bet is that is slightly more performant.